subsys/bluetooth/controller/ll_sw/ull_df_internal.h

```diff
@@ -20,4 +20,6 @@ void ull_df_rx_iq_report_alloc(uint8_t max);
 /* Initialized DF sync configuration. */
 void ull_df_sync_cfg_init(struct lll_df_sync *cfg);
 /* Returns information if CTE sampling for periodic sync is requested to disable. */
-uint8_t ull_df_sync_cfg_is_disabled_or_requested_to_disable(struct lll_df_sync *df_cfg);
+bool ull_df_sync_cfg_is_not_enabled(struct lll_df_sync *df_cfg);
+/* Returns information if CTE sampling for a connection is not enabled. */
+bool ull_df_conn_cfg_is_not_enabled(struct lll_df_conn_rx_cfg *rx_cfg);
```

Comment:
Given that these are `get`ters, the arguments could and should be made `const`.

Reply:
These can't be changed to `const` because the functions use e.g. 
```
static inline void *dbuf_peek(struct dbuf_hdr *hdr)
{
	return &hdr->data[hdr->last * hdr->elem_size];
}
```
that can't have `const struct dbuf_hdr *hdr` due to return `void*`. That would limit the usage of the `dbuf_peek` funcion.

Reply:
I see :)